playbooks/streisand-client.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
How about this: I use `test-client.yml` since its generic to end-to-end testing and not specific to vagrant. The vagrant specific pieces stay in `vagrant.yml`. 

The next immediate question is "If it's for testing, why isn't it in the `test/` directory with Corban's CI work". The answer to _that_ is that I don't want to use the inventory and other settings files that he has in the top level of that dir. I think a good follow-up issue would be to move some of that to `test/ci` and then make `test/vagrant` and put `test-client.yml` in there.

Sounds good to you folks?